pmd-core/src/main/java/net/sourceforge/pmd/cli/PMDCommandLineInterface.java

```diff
@@ -27,6 +27,7 @@
     public static final String NO_EXIT_AFTER_RUN = "net.sourceforge.pmd.cli.noExit";
     public static final String STATUS_CODE_PROPERTY = "net.sourceforge.pmd.cli.status";
 
+    public static final int NO_ERRORS_STATUS = 0;
```

Comment:
About the API footprint - I'd declare this as private, so that we don't introduce another public global constant, that we later on would need to deprecate in order to make them private....